addons/binding/org.openhab.binding.denonmarantz/src/main/java/org/openhab/binding/denonmarantz/internal/xml/adapters/VolumeAdapter.java

```diff
@@ -12,7 +12,7 @@
 
 import javax.xml.bind.annotation.adapters.XmlAdapter;
 
-import org.openhab.binding.denonmarantz.DenonMarantzBindingConstants;
+import org.openhab.binding.denonmarantz.internal.DenonMarantzBindingConstants;
```

Comment:
Can we maybe reafactor this class to use `DenonMarantzBindingConstants` as static import only?